Anatoly Shlom/flex/css/flex.less

```diff
@@ -0,0 +1,193 @@
+
+/* Приводим браузерные стили в порядок */
+@import url('normalize.css');
+
+/* Добавляем необходимый шрифт*/
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700');
+* {
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+    color: #ffffff;
+    padding: 0;
+    margin: 0;
+}
+
+/*Добавляем фон и немножко кросбраузерности)*/
+html { 
+    background: url("../img/bg.png") no-repeat center center fixed; 
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+
+/*оформляем сам виджет*/
+.widget {
+    background: no-repeat;
+    background-image: url("../img/main_bg.png");
+    background-size: cover; 
+    margin: 7% 11% 7% 11%;
+    padding: 10% 0 0 0;
+}
+
+/*верхняя часть виджета*/
+@media (max-width: 600px) {
+    .widget-data {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+    }
+}
+@media only screen and (min-width: 600px) {
+    .widget-data {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        align-items: flex-start;
+        margin:0 10% 0 0;
+    }
+}
+/*секция с текущей датой*/    
+@media only screen and (min-width: 600px) {
+    .current-date {
+        margin: 20px 20px 0 0 ;
+    }
+}
+.current-date__week-day {
+    font-size: 30px;
+}
+.current-date__day {
+    font-size: 14px;
+}
+
+/*секция с температурой,локацией,ветром*/
+.weather-block {
+    display: table-cell;
+    text-align: center;
+}
+.weather-block__temperature {
+    font-size: 92px;
+    font-weight: 700;
+}
+.weather-block__location {
+    font-size: 22px;
+}
+.wind-block {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-end;
+}
+
+.wind-block__speed {
+    font-size: 29px;
+    font-weight: 400;
+}
+.wind-block__speed-format {
```

Comment:
Preprocessor allows you to write it like so:

```
.wind-block__speed {
    font-size: 29px;
    font-weight: 400;

    &__speed {
         ...
     }

    &__speed-format {
        ...
    }
}
```

By this way you wouldn't rename all styles when you decided to change name of the block

Reply:
Thats really awesome feature!! Thank you!!